cobbler.spec

```diff
@@ -213,7 +213,7 @@ Requires:       python%{python3_pkgversion}-requests
 Requires:       python%{python3_pkgversion}-simplejson
 Requires:       python%{python3_pkgversion}-tornado
 Requires:       python%{python3_pkgversion}-distro
-Recommends:     python%{python3_pkgversion}-ldap3
+Requires:       python%{python3_pkgversion}-ldap3
```

Comment:
Well, this would break SLES right now. What was the reason for having it like this? RHEL is not using `Recommends`?

Reply:
I would have to look that up. But when using LDAP as authentication backend, first you'll need to install python3-ldap, otherwise authentication won't work. That's why it's good to have that as a requirement.

Reply:
Do you want to install all packages which provide all kind of authentication mechanisms, even if you do not use them?

Reply:
Yeah, or at least provide documentation which packages need to be installed. On RHEL for example the package name is pyhon3-ldap, not python3-ldap3, you wouldn't get that from the Recommends.

Reply:
Then we need to wrap that in an if. @nodeg

Reply:
I defined a macro for `python3-ldap` and wrapped the requirements/recommends in an if statement.